### PR TITLE
Ds 445 split bundle

### PR DESCRIPTION
**Pull request by kielbasa-elp**

### CODE REVIEW

File: packages/design-system/rollup.config.js [hunk 1]
```diff
@@ -12,57 +12,83 @@ import resolve from "@rollup/plugin-node-resolve";
...
+const config = [bundles, ...types];
+
 export default config;
+
+function preparePaths(names) {
+  return names.map((component) => ({
+    js: "./src/components/" + component + "/index.ts",
+    types: "dist/components/" + component + "/index.d.ts",
+  }));
+}
```
Comment (piotrekzawada) [inline packages/design-system/rollup.config.js:94]: extract to helper function

File: packages/design-system/src/components/Carousel/index.ts
Comment (piotrekzawada) [inline packages/design-system/src/components/Carousel/index.ts:1]: fix linting

**Comment by piotrekzawada** [comment]
@kielbasa-elp please add instruction to readme

File: packages/design-system/rollup.config.js [hunk 2]
```diff
@@ -12,57 +12,83 @@ import resolve from "@rollup/plugin-node-resolve";
...
-    input: "dist/index.d.ts",
-    output: [{ file: "dist/index.d.ts", format: "es" }],
+    input: ["dist/index.d.ts", ...paths.map((path) => path.types)],
+    output: [{ dir: "dist", format: "es" }],
     external: [/\.css$/],
     plugins: [
+      multiInput.default({
+        transformOutputPath: (output) => {
+          const segments = output.split("/");
+
```
Comment (zuzannanap) [inline packages/design-system/rollup.config.js:73]: maybe better without this empty line?

File: packages/design-system/src/components/Carousel/index.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Carousel from './Carousel'
+
+import type {CarouselProps,SlidesCarouselProps} from "./Carousel.interface";
+
+export {Carousel}
+
+export type {CarouselProps,SlidesCarouselProps}
```
Comment (zuzannanap) [inline packages/design-system/src/components/Carousel/index.ts:7]: add empty line at the end

File: packages/design-system/src/components/DatePicker/index.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import DatePicker from './DatePicker'
+import type {DatePickerProps} from "./DatePicker.interface";
+
+export {DatePicker}
+export type {DatePickerProps}
```
Comment (zuzannanap) [inline packages/design-system/src/components/DatePicker/index.ts:5]: same

File: packages/design-system/src/index.ts [hunk 5]
```diff
@@ -99,15 +99,10 @@ import { Input } from "./components/Input";
 import { InputProps } from "./components/Input/Input.interface";
 import InputCode from "./components/InputCode";
 import { InputCodeProps } from "./components/InputCode/InputCode.interface";
-import DatePicker from "./components/DatePicker";
-import { DatePickerProps } from "./components/DatePicker/DatePicker.interface";
+
+
```
Comment (zuzannanap) [inline packages/design-system/src/index.ts:103]: 2 empty lines to remove

**Review by zuzannanap — CHANGES_REQUESTED** [review]

**Pull request merged**